### PR TITLE
Create todo.15m.rb

### PR DESCRIPTION
**Pull request by Srdgh**
Bitbar plugin showing your todo list with color-coding.

### CODE REVIEW

File: Lifestyle/todo.15m.rb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+# Customise here: label color-code (colors optimised for dark theme menubar)
+@Labels = { 
+    "+Urgent"=>"red",
+    ""=>"orange",
+    ""=>"yellow",
+    ""=>"green",
+    ""=>"cadetblue",
+    ""=>"purple",
+    ""=>"violet"
```
Comment (keithamus) [inline Lifestyle/todo.15m.rb]: It would be good to provide some sensible defaults here.

File: Lifestyle/todo.15m.rb [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+# coding: utf-8
+
+# <bitbar.title>TodoColor</bitbar.title>
+# <bitbar.version>v0.1</bitbar.version>
+# <bitbar.author>Srdgh</bitbar.author>
+# <bitbar.author.github>Srdgh</bitbar.author.github>
+# <bitbar.desc>Todo list color-coded</bitbar.desc>
+# <bitbar.dependencies>ruby</bitbar.dependencies>
+
+@todo_file = File.open("$HOME/.todo") #todo file path
```
Comment (keithamus) [inline Lifestyle/todo.15m.rb]:
```rb
@todo_file = File.open("#{ENV['HOME']}/.todo") #todo file path
```

Would work better here.

**Comment by keithamus** [comment]
@Srdgh I've made some small notes, it'd be great if you could address these, then I can merge :smile:

**Comment by Srdgh** [comment]
@keithamus Thanks -- done. Also added some tweaks.

**Comment by keithamus** [comment]
:+1:

**Comment by keithamus** [comment]
Actually I merged that a bit hastily - @Srdgh could you make a new PR adding a screenshot to the metadata please?

**Comment by Srdgh** [comment]
OK -- doing so now.

**Comment by Srdgh** [comment]
(Done)

**Pull request merged**